package/Makefile.linux

```diff
@@ -68,6 +68,7 @@ ground_package_os_specific: ground_package_common
 ifeq ($(EXPORT_SYMBOLS), YES)
 	$(V1) $(ROOT_DIR)/package/linux_extract_debug_symbols.sh $(DUMP_SYMBOLS_TOOL) $(PACKAGE_DIR)/$(GCS_PACKAGE_NAME)
 endif
+	$(V1) cp -v $(ROOT_DIR)/package/linux/deb/_package.udev $(PACKAGE_DIR)/rules.udev
```

Comment:
So I don't have the branding down here (I don't think), so I can't construct a complete name.  I was thinking docs would instruct user what to do?

The deb attempts to do the right thing.